### ISSUE TITLE
Redirect users to /trends by default

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#476]
***Thank*** you for your feature request - we love each and every one :)

**Is your feature request related to a problem? Please describe.**
/trends is the best way to do random data discovery. It's also our most popular page (closely followed by /events, but I think that's less interesting) 
![image](https://user-images.githubusercontent.com/1727427/78666137-d035bc00-78ce-11ea-88ee-75d7b1542958.png)

**Describe the solution you'd like**
When a user goes to their instance the first time, they should be redirected to /trends. This probably means the current `/` needs to become `/dashboard`. This will save a lot of people a click, and immediately gets them exploring our data.

**Comment by BElluu** [comment]
Hello!
Can I take care of this task? :)

**Comment by jamesefhawkins** [comment]
That’d be awesome! Of course

**Comment by timgl** [comment]
@BElluu Hey! Did you manage to get anywhere with this? Let me know if you need help with anything :)

**Comment by BElluu** [comment]
@timgl Hey! Sorry but I didn't have time for that because I had  a lot of preparation before Easter. I will look at this tomorrow :) Thanks for showing interest

**Comment by BElluu** [comment]
Hi!
I had minor problems running the application as described in the development section, but today I will try to deal with them and start carrying out the task.

**Comment by BElluu** [comment]
Hi everybody,
@timgl @jamesefhawkins 
someone can give me advice? I tried to prepare my env to development but I had problem with  the ninth point in the instructions. When I run `DEBUG=1 python3 manage.py migrate` I have disturbing output

```
(env)  ⚙ belluu@belluu-sys  ~/programming/posthog   master  DEBUG=1 python3 manage.py migrate
Traceback (most recent call last):
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 220, in ensure_connection
    self.connect()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 197, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/postgresql/base.py", line 185, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/psycopg2/__init__.py", line 126, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: fe_sendauth: no password supplied


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    main()
  File "manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/core/management/base.py", line 328, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/core/management/base.py", line 369, in execute
    output = self.handle(*args, **options)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 86, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/migrations/loader.py", line 49, in __init__
    self.build_graph()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/migrations/loader.py", line 212, in build_graph
    self.applied_migrations = recorder.applied_migrations()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/migrations/recorder.py", line 76, in applied_migrations
    if self.has_table():
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/migrations/recorder.py", line 56, in has_table
    return self.Migration._meta.db_table in self.connection.introspection.table_names(self.connection.cursor())
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 260, in cursor
    return self._cursor()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 236, in _cursor
    self.ensure_connection()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 220, in ensure_connection
    self.connect()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 220, in ensure_connection
    self.connect()
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/base/base.py", line 197, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/django/db/backends/postgresql/base.py", line 185, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/home/belluu/programming/posthog/env/lib/python3.6/site-packages/psycopg2/__init__.py", line 126, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
django.db.utils.OperationalError: fe_sendauth: no password supplied

```
This is related to the database. It looks like the application cannot authenticate with the database. The application uses the database called "posthog", I created one. What is the user using to connect to this database? - postgres? posthog? What about password?

**Comment by timgl** [comment]
By default we pass `postgres://localhost:5432/posthog` as the DATABASE_URL. I'm not sure what your local username/password is (how did you create this postgres db?) but you could do something like `postgres://username:password@localhost:5432/posthog` if you do have auth details

**Comment by BElluu** [comment]
@timgl 
I created database using PGAdmin4. I have database with login postgres and my password. When I try do step `DEBUG=1 python3 manage.py migrate` I got this error stack.
I have read `fe_sendauth: no password supplied` means attempting to log in with an empty password to server, but PostgreSQL does not normally allow empty user passwords

![image](https://user-images.githubusercontent.com/11333925/79378820-420ca600-7f5e-11ea-956b-6ded728f878e.png)

**Comment by timgl** [comment]
Have you set `DATABASE_URL=postgres://username:password@localhost:5432/posthog` ?

**Comment by BElluu** [comment]
@timgl Please tell me, where should I set this? I do not see step with that in instruction :)

**Comment by timgl** [comment]
Same way as DEBUG, so something like `DATABASE_URL=postgres://username:password@localhost:5432/posthog python manage.py migrate`

It's worth setting it in your `~/.bashrc` or `~/.zshrc` as well, as `export DATABASE_URL=postgres://username:password@localhost:5432/posthog`

**Comment by BElluu** [comment]
@timgl that's works! Thank you! I think it is worth adding this to the instructions for Linux users

Now that the application is working, I will take care of this task tomorrow. Again thank you very much! :)

**Comment by eLRuLL** [comment]
I hope this isn't an inconvenience but I was playing with the application and made a fix for this on #656 please let me know what you guys think
